.github/workflows/generate-summary.yaml

```diff
@@ -261,6 +261,37 @@ jobs:
               body: 'Bisecting with #${{ needs.generate-issues.outputs.new_issue_num }}'
             })
 
+      - name: Retrieve cache
+        id: retrieve-cache
+        uses: actions/cache@v3
+        with:
+          path: |
+            .git
+            binutils
+            dejagnu
+            gcc
+            gdb
+            glibc
+            musl
+            newlib
+            pk
+            qemu
+          key: submodules
+
+      - name: Initalize gcc
+        if: steps.retrieve-cache.outputs.cache-hit != 'true'
+        run: |
+          rm -rf gcc
+          git clone git://gcc.gnu.org/git/gcc.git
+          git submodule update --recursive --progress --recommend-shallow
```

Comment:
Since we only use GCC, I think we can remove this line.

Reply:
testing regenerate: https://github.com/patrick-rivos/riscv-gnu-toolchain/actions/runs/6090823639
other test: https://github.com/ewlu/riscv-gnu-toolchain/actions/runs/6090811474